mojo-executor/pom.xml

```diff
@@ -59,5 +59,13 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+        </dependency>
     </dependencies>
```

Comment:
Given the difficulty in figuring out the cause in e.g. #32 I thought it useful to add some more (debug) logging